src/main/java/org/prebid/server/currency/CurrencyConversionService.java

```diff
@@ -124,11 +124,12 @@ public ZonedDateTime getLastUpdated() {
     }
 
     /**
-     * Converts price from bidCurrency to adServerCurrency using rates defined in request or if absent, from
-     * latest currency rates. Throws {@link PreBidException} in case conversion is not possible.
+     * Converts price from bidCurrency to adServerCurrency using rates and usepbsrates flag defined in request.
+     * If usepbsrates is true it takes rates from prebid server, if false from request.
+     * or if absent, from latest currency rates. Throws {@link PreBidException} in case conversion is not possible.
      */
     public BigDecimal convertCurrency(BigDecimal price, Map<String, Map<String, BigDecimal>> requestCurrencyRates,
-                                      String adServerCurrency, String bidCurrency) {
+                                      String adServerCurrency, String bidCurrency, boolean usepbsrates) {
```

Comment:
boolean -> Boolean

And a bit lower you can use **BooleanUtils.isTrue** to use appropriate logic. And it be good approach if you add comments, that false is by default 
isTrue = true only of true. So else case will be for null and false.